zh.perl6intro.adoc

```diff
@@ -714,6 +715,10 @@ say %capitals.WHAT;
 my Int %country-codes = (UK => 44, Germany => 49);
 say %country-codes;
 say %country-codes.WHAT;
+
+my Str %student-name{Int} = (1 => 'Alex', 2 => 'Mary');
+say %student-name;
+say %student-name.WHAT;
 ----
```

Comment:
Add non-string keys hashes, it does not written in the origin(English) ver., but I think it is worthy.
Also mention in https://github.com/perl6/doc/issues/1588
@hankache